Nightly build for IgaApplication fails

As you might have recognized the nightly build for the IgaApplication failed. 

`Failed Applications : 1
IgaApplication --> KratosIgaApplication.dir
Compiling Errors    : 3
/home/ubuntu/Kratos/applications/IgaApplication/custom_python/add_custom_utilities_to_python.cpp:266:5: error: no matching function for call to ‘pybind11::class_<ANurbs::SurfaceGeometryBase<Kratos::array_1d<double, 1ul> >, std::shared_ptr<ANurbs::SurfaceGeometryBase<Kratos::array_1d<double, 1ul> > > >::def(const char [7], <unresolved overloaded function type>, pybind11::arg, pybind11::arg)’
/home/ubuntu/Kratos/applications/IgaApplication/custom_python/add_custom_utilities_to_python.cpp:266:5: error: no matching function for call to ‘pybind11::class_<ANurbs::SurfaceGeometryBase<Kratos::array_1d<double, 2ul> >, std::shared_ptr<ANurbs::SurfaceGeometryBase<Kratos::array_1d<double, 2ul> > > >::def(const char [7], <unresolved overloaded function type>, pybind11::arg, pybind11::arg)’
/home/ubuntu/Kratos/applications/IgaApplication/custom_python/add_custom_utilities_to_python.cpp:266:5: error: no matching function for call to ‘pybind11::class_<ANurbs::SurfaceGeometryBase<Kratos::array_1d<double, 3ul> >, std::shared_ptr<ANurbs::SurfaceGeometryBase<Kratos::array_1d<double, 3ul> > > >::def(const char [7], <unresolved overloaded function type>, pybind11::arg, pybind11::arg)’`

I compiled on my own machine, but there it seems to work, correctly. Maybe someone else can check this.

--- a/applications/IgaApplication/custom_python/add_custom_utilities_to_python.cpp
+++ b/applications/IgaApplication/custom_python/add_custom_utilities_to_python.cpp
@@ -301,7 +301,10 @@ void RegisterSurfaceGeometryBase(
             "IndexU"_a,
             "IndexV"_a,
             "Value"_a)
-        .def("Weight", &Type::Weight,
+        .def("Weight", (double (Type::*)(const int) const) &Type::Weight,
+            "Index"_a)
+        .def("Weight", (double (Type::*)(const int, const int) const)
+            &Type::Weight,
             "IndexU"_a,
             "IndexV"_a)
         .def("SetWeight", (void (Type::*)(const int, const double))

